<?xml version="1.0"?>
<launch>

  <arg name="motor_count" default="1" />
  <arg name="joint_counts" default="[10]" />
  <arg name="home_vals_config" default="$(find bata_base)/config/home_vals.yaml" />

  <param name="motor_count" value="$(arg motor_count)" />
  <rosparam param="joint_counts"  subst_value="True">$(arg joint_counts)</rosparam>
  <param name="robot_description" command="$(find xacro)/xacro --inorder
                       '$(find bata_description)/robots/bata.urdf.xacro'
                        joint_counts:=$(arg joint_counts)"/>
  <rosparam file="$(arg home_vals_config)" command="load" />

  <node pkg="robot_state_publisher" type="robot_state_publisher" name="state_publisher"/>

  <node pkg="bata_base" type="bata_interface.py" name="bata_interface" output="screen"/>

  <node pkg="bata_base" type="bata_path_controller.py" name="bata_path_controller" output="screen"/>

  <!--  <include file="$(find realsense2_camera)/launch/rs_camera.launch"/> -->

</launch>
